pkg/webhook/validate.go

```diff
@@ -34,8 +35,8 @@ func validateSriovOperatorConfig(cr *sriovnetworkv1.SriovOperatorConfig, operati
 	glog.V(2).Infof("validateSriovOperatorConfig: %v", cr)
 	var warnings []string
 
-	if cr.GetName() == "default" {
-		if operation == "DELETE" {
+	if cr.GetName() == utils.DefaultConfigName {
+		if operation == Delete {
```

Comment:
no definition in v1 package ? i see operation is of type v1.Operation

Reply:
Done